client/src/ui/deckbuilder/DeckBuilderMain.vue

```diff
@@ -30,22 +30,33 @@
           :horizontal="horizontal"
           />
     </div>
-    <a
-        :href="exportedDeck"
-        download="r38export.dek"
-        class="exportButton"
-        :hidden="!deck || horizontal"
-        >
-      Export deck
-    </a>
-    <a
-        :href="exportedBinder"
-        download="r38export.dek"
-        class="exportButton"
-        :hidden="!deck || horizontal"
-        >
-      Export binder
-    </a>
+    <div
+        class="exportButtons">
+      <a
+          :href="exportedDecksZip"
```

Comment:
I'm worried about recomputing the deck zip every time something changes (which, now that the deck builder is integrated with the overall replay layout will be more often than before).

Could we instead have a single "export" button -- when the user clicks it, we generate a mini-menu of the three options and only build the deck XML/decks zip then (maybe in the onCreate() method of the subcomponent?)

Happy to help here; see `client/src/ui/replay/controls_row/TimelineButton.vue` for an example of creating such a floating menu, especially the methods section.